test/entity.ts

```diff
@@ -1107,426 +1111,6 @@ describe('entity', () => {
     });
 
     it('should respect excludeFromIndexes', () => {
```

Comment:
The fixed constants defined below are now going to get reused for the new buildEntityProto function so we pull them out into a separate file so that they can be used here and for `buildEntityProto` tests as well.